Resources/public/style/httplug.css

```diff
@@ -80,6 +80,11 @@
     font-size: 12px;
 }
 
+.httplug-duration {
+    min-width: 6ch;
```

Comment:
hm. Interesting. Thanks. 

What if it takes more then one sec? "1000 ms" That is 7 chars, right? Will we get a line break? If so, change this to `7ch` and we will be all happy.